src/common/Elsa.Testing.Shared.Integration/WorkflowExtensions.cs

```diff
@@ -0,0 +1,88 @@
+﻿using Elsa.Extensions;
+using Elsa.Features.Services;
+using Elsa.Mediator.Contracts;
+using Elsa.Workflows;
+using Elsa.Workflows.Notifications;
+using Elsa.Workflows.Runtime;
+using Elsa.Workflows.Runtime.Contracts;
+using Elsa.Workflows.Runtime.Requests;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace Elsa.Testing.Shared;
+
+public static class WorkflowExtensions
+{
+    public static async Task<WorkflowFinished?> DispatchWorkflowAndRunToCompletion(

```

Comment:
That's a really cool and clever way of setting up a test host, very flexible. Taking this in 👍🏻 👍🏻